gramps/plugins/tool/reorderids.py

```diff
@@ -34,207 +35,516 @@
 from gramps.gen.const import GRAMPS_LOCALE as glocale
 _ = glocale.translation.gettext
 
+from gi.repository import Gtk, Gdk
+
 #------------------------------------------------------------------------
 #
-# Gramps modules
+# GRAMPS modules
```

Comment:
Note that the project name is no longer in Uppercase since the 1st Gramps 4.0 release at least see https://sourceforge.net/p/gramps/mailman/message/35025006/